PelotonEppSdk/Enums/ClientAuthTokenAuthorizationType.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PelotonEppSdk.Enums
+{
+    public enum ClientAuthTokenAuthorizationType
+    {
+        CreditCard = 1,
```

Comment:
Card = Card Interactions (Add Card / Update Card / Delete Card)
BankAccount = BankAccount Interactions (Add Bank Account / Update Bank Account / Delete Bank Account)

This is all we are going to support with this release the others should be deleted for now.